src/ServicePulse.Host/vue/src/views/MessageView.vue

```diff
@@ -0,0 +1,119 @@
+<script setup>
+import { ref, onMounted } from "vue";
+import { useRoute } from "vue-router";
+import { useFetchFromServiceControl } from "../composables/serviceServiceControlUrls";
+import NoData from "../components/NoData.vue";
+import TimeSince from "../components/TimeSince.vue";
+
+const route = useRoute();
+const id = route.params.id;
+const failedMessage = ref({});
+
+function loadFailedMessage() {
+  return useFetchFromServiceControl("errors/last/" + id)
+    .then((response) => {
+      if (response.status === 404) {
+        failedMessage.value = { notFound: true };
+      } else {
+        failedMessage.value = { error: true };
+      }
+      return response.json();
+    })
+    .then((data) => {
+      var message = data;
```

Comment:
What are these values if the response is a 404? Will this fail?

Reply:
These values are null because if the response status is 404, it skips the `then` that assigns the data values.  However, I have not been able to generate an error status to test if that gets caught.  This does assume that if the status is anything other than a 404 it sets error to true and should be caught, unless the status is `200 ok` and then it should assign the data to the object and overwrite the failedMessage.value.  I might need to update this to be more specific about `200 ok` status.
![image](https://user-images.githubusercontent.com/87037242/220262089-4d7bf3a0-c167-46b7-9940-c71056d386e9.png)
![image](https://user-images.githubusercontent.com/87037242/220262118-37c3146f-a4a3-471d-8085-1b4ac7b0849b.png)